server/controller/Users/UserQueries.js

```diff
@@ -0,0 +1,46 @@
+const dbconnection = require('../../database/db_connection');
+
+const mysql = require("mysql");
+    // TODO: Global database config
+    const dbConfig = mysql.createConnection({
+        user: "b4bb330a3f009b",
+        host: "eu-cdbr-west-01.cleardb.com",
+        password: "4be56a74",
+        database: "heroku_1fcc54407741c0e",
+      });
```

Comment:
This config has been set up in the global file.

Reply:
Yesterday, I require him to demo for me, then the config has to put in there to test, I told him has to refactor to match with the global config @Dev01-vinamr-Ken